moped-database/migrations/1665693611607_add_component_geometry_tables/up.sql

```diff
@@ -0,0 +1,197 @@
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Pavement Marking' and component_subtype = 'Two-stage Bike Turn Queue';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Pavement Marking' and component_subtype = 'Sharrows';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Pavement Marking' and component_subtype = 'Sharrows (Wide Curb Lane)';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Signage' and component_subtype is null;
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Continental';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Creative';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Crosswalk' and component_subtype = 'Raised';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Sidewalk' and component_subtype = 'Ramps';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Lighting' and component_subtype = 'Street lighting';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Lighting' and component_subtype = 'Intersection lighting';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Transit' and component_subtype = 'Bus stop';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments') where component_name = 'Project Extent - Generic' and component_subtype is null;
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Signage' and component_subtype = 'RRFB';
+update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Intersection' and component_subtype = 'Roundabout';
```

Comment:
@frankhereford we have an additional component making it's way into the DB in v1.11:
https://github.com/cityofaustin/atd-moped/pull/844/files

This should cover it:

```sql
update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_street_segments'  ) where component_name = 'Intersection' and component_subtype = 'Improvement (linear)';

update moped_components set feature_layer_id = (select id from feature_layers where internal_table = 'feature_intersections'  ) where component_name = 'Intersection' and component_subtype = 'Improvement';
```

Reply:
Thank you @johnclary - both for observing that these would be missing and then for even providing the `update` statements!